content/docs/reference/routes/public-access.mdx

```diff
@@ -16,9 +16,24 @@ import TabItem from '@theme/TabItem';
 
 ## Summary
 
-**Use with caution:** **Public Access** allows all requests for a given route, bypassing authentication and authorization. Suitable for publicly exposed web services.
+:::caution
 
-If this setting is enabled, no whitelists (for example, Allowed Users) should be provided in this route.
+Because the **Public Access** setting bypasses authentication and authorization checks, you should only enable it for publicly exposed web services.
+
+:::
+
+**Public Access** allows all requests to a given route and bypasses authentication and authorization. If you enable this setting, no whitelists should be provided for the route (for example, **Allowed Users**).
```

Comment:
I think "Allowed Users" was a setting that existed before PPL. I'm not sure that it's still supported, but even if it is I don't think it's documented, so I don't think it makes sense to refer to it here.

Maybe something like this instead?
```suggestion
**Public Access** allows all requests to a given route and bypasses authentication and authorization. If you enable this setting, no other policy should be provided for the route.
```